guides/doc-Managing_Content/master.adoc

```diff
@@ -43,6 +43,10 @@ include::common/assembly_managing-content-views.adoc[leveloffset=+1]
 
 include::common/assembly_synchronizing-content-between-servers.adoc[leveloffset=+1]
 
+ifdef::satellite[]
+include::common/modules/proc_using-project-as-a-source-of-content-for-imagebuilder.adoc[leveloffset=+1]
```

Comment:
I do not recommend putting this section at the top level of the navigation. Can we find a better place for it?